content/05.discussion.md

```diff
@@ -12,22 +12,22 @@ We then compared this to the authorship statistics from _Nature_'s papers and a
 We first looked at possible gender differences in quotes and found a large, but decreasing, gender gap when compared to the general population in all but one article type.
 Additionally, this result was consistent in articles written by journalists predicted to be women or men.
 We found that one column, "Career Feature", has an equal number of quotes from both genders, showing that gender parity is possible in science journalism.
-This finding, coupled with the near-equal number of articles written by journalists predicted to be women or men, argues for more diversity in topical coverage.
-"Career Feature" articles highlight current topics relevant to working scientists and frequently highlight systemic issues within the scientific environment.
+This finding, coupled with the near equal number of articles written by male and female predicted journalists, argues for more diversity in topical coverage. 
```

Comment:
As I'm thinking about this, is it more appropriate for this to be "journalists predicted to be women or men"? I may have introduced an incorrect change. We are probably predicting gender more than sex.